plugins/node/instrumentation-fs/test/index.test.ts

```diff
@@ -0,0 +1,366 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { context, trace, SpanStatusCode, SpanKind } from '@opentelemetry/api';
+import { AsyncHooksContextManager } from '@opentelemetry/context-async-hooks';
+import {
+  BasicTracerProvider,
+  InMemorySpanExporter,
+  ReadableSpan,
+  SimpleSpanProcessor,
+} from '@opentelemetry/sdk-trace-base';
+import * as assert from 'assert';
+import Instrumentation from '../src';
+import { Resource } from '@opentelemetry/resources';
+import { SemanticResourceAttributes as ResourceAttributesSC } from '@opentelemetry/semantic-conventions';
+import { JaegerExporter } from '@opentelemetry/exporter-jaeger';
+import * as sinon from 'sinon';
+import type * as FSType from 'fs';
+import tests, { TestCase, TestCreator } from './definitions';
+import type { FMember, FPMember } from '../src/types';
+
+const supportsPromises = parseInt(process.versions.node.split('.')[0]) >= 12;
```

Comment:
in `instrumentation.ts` the check for > 8.

Reply:
Unified those.